.pre-commit-config.yaml

```diff
@@ -1,5 +1,26 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
+# See https://pre-commit.ci/#configuration
+# See https://github.com/scientific-python/cookie#sp-repo-review
+
+ci:
+  autofix_prs: false
+  autoupdate_commit_msg: "chore: update pre-commit hooks"
+
+
+# Alphabetised, for lack of a better order.
+files: |
+    (?x)(
+        benchmarks\/.+\.py|
+        docs\/.+\.py|
+        esmf_regrid\/.+\.py|
+        noxfile\.py|
+        pyproject\.toml|
+        setup\.py|
+        src\/.+\.py
```

Comment:
This doesn't seen necessary since we don't have a src file. Although it doesn't seem like it is a problem since its not causing errors.